frontend/www/js/omegaup/components/common/Navbar.vue

```diff
@@ -61,10 +61,28 @@
             <a href="/arena/" data-nav-contests-arena>{{ T.wordsContests }}</a>
           </li>
           <li
-            class="nav-courses"
+            class="dropdown nav-courses"
             v-bind:class="{ active: navbarSection === 'courses' }"
+            v-if="isLoggedIn"
           >
-            <a href="/course/">{{ T.navCourses }}</a>
+            <a class="dropdown-toogle" data-toggle="dropdown" data-nav-courses
+              ><span>{{ T.navCourses }}</span
+              ><span class="caret"></span>
+            </a>
+            <ul class="dropdown-menu">
+              <li>
+                <a href="/course" data-nav-courses-all>
```

Comment:
nit:

```suggestion
                <a href="/course/" data-nav-courses-all>
```

siempre que se pueda, los URLs deben terminar con `/`. la razón es que las reglas de nginx siempre agregan ese `/` al final mediante una redirección, entonces así se evita un roundtrip al servidor.